### DISCUSSION TITLE
반응형 비율 유지 안됨

### DISCUSSION THREAD

**Discussion by da-in** [Yangjaecheon-Otter-Guardians/simple-thumbnail#134] [Bugs]
### Bugs Description

```bash
화면 사이즈가 변할 때(웹에서 브라우저 가로 사이즈 바꾸면) break point에 따라 미리보기 이미지의 높이는 변하는데, 폭이 변하지 않습니다.
```

### BDD TASK

```bash
- when: 화면 사이즈가 변하면
- then: 미리보기 영역의 높이와 함께 폭도 변하여, 비율이 유지되어야합니다.
```

**Comment by caseBread** [comment]
오 저도 재현되었습니다 상세한 큐에이 감사합니다. 🙇‍♂️

모바일일 때 설정한 레이아웃 대비 실제 레이아웃이 제대로 안나오게 되는..거겠군요

즉 사용자 의도대로 결과물이 안나오게 되는 부분이라 우선순위 좀 높게 잡는게 좋을까요?

**Reply by caseBread** [reply]
아 이거 새로고침하면 비율이 바뀌긴 하는군요

**Reply by da-in** [reply]
네네! 그래도 말씀하신 것 처럼 우선순위를 높게 잡아도 좋을 것 같습니다 👍🏻
